Blog/App_Start/Startup.cs

```diff
@@ -10,12 +16,70 @@ public class Startup
     {
         public void Configuration(IAppBuilder app)
         {
+            CreateUserRoles();
```

Comment:
To odpala sie przy kazdym starcie aplikacji, powinno byc raczej podczas tworzenia bazy.